sources/packages/backend/libs/integrations/src/services/disbursement-schedule/e-cert-generation.service.ts

```diff
@@ -322,4 +323,25 @@ export class ECertGenerationService {
       },
     });
   }
+
+  /**
+   * Get the lifetime maximums of CSLP and is used in the e-Cert generation.
```

Comment:
Please review the comment, "and is used in" seems to be not providing a clear message.